engine/src/main/java/org/terasology/rendering/openvrprovider/OSValidator.java

```diff
@@ -0,0 +1,61 @@
+package org.terasology.rendering.openvrprovider;
+
+/* Mainly used to locate the OpenVR library on different platforms. */
+public class OSValidator {
```

Comment:
Good catch. Yep there is indeed a `org.terasology.utilities` package which even has an existing `NativeHelper`, it might be an idea to look at enhancing that :-)